src/OrchardCore/OrchardCore.Abstractions/Json/Dynamic/JsonDynamicObject.cs

```diff
@@ -95,6 +97,14 @@ public bool Remove(string key)
             return null;
         }
 
+        foreach (var handler in JsonDynamicConfigurations.ValueHandlers)
```

Comment:
Added a static collection to handle dynamic values for custom development environments